Dockerfile

```
@@ -0,0 +1,19 @@
+WORKDIR /app
+
+# Copy package files and install
+COPY package.json package-lock.json ./
+RUN npm ci --omit=dev
+
+# Copy source
+COPY src/ ./src/
+
+# Don't run as root
+RUN addgroup --system --gid 1001 keeper
+RUN adduser --system --uid 1001 keeper
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)